prometheus.yml

```diff
@@ -0,0 +1,20 @@
+global:
```

Comment:
Since I'm using prometheus mainly at the moment for development purposes, I'm not completely certain if this configuration file should be in the root of the project. If this is ok I'll leave it here, otherwise suggestions are welcome.

Reply:
I think we shouldn't have too many files in the root, but I don't know what a better location is, so I think we can put it there for now and we can always move it later on.